### DISCUSSION TITLE
`freezeCheck` exception with SDKMAN

### DISCUSSION THREAD

**Discussion by sualeh** [jreleaser/jreleaser#845] [Q&A]
@aalmiray - I have started getting a [freezeCheck exception with SDKMAN](https://github.com/schemacrawler/SchemaCrawler-Installers/runs/6885445413?check_suite_focus=true#step:4:341) in my [JReleaser process](https://github.com/schemacrawler/SchemaCrawler-Installers/blob/master/jreleaser.yml). Please could you help me spot what I need to change? It was working a couple of days ago.

**Comment by aalmiray** [comment]
@sualeh please post the full stack trace you've got

**Reply by sualeh** [reply]
@aalmiray - it is in the [link](https://github.com/schemacrawler/SchemaCrawler-Installers/runs/6885445413?check_suite_focus=true#step:4:341) above. Also, that link points to the GitHub Action that has the [trace log](https://github.com/schemacrawler/SchemaCrawler-Installers/actions/runs/2496937241).

**Reply by aalmiray** [reply]
I've taken care of the problem. Next `early-access` build will have the fix. You'll have to wait til https://github.com/jreleaser/jreleaser/actions/runs/2497073400 is done.
